### DISCUSSION TITLE
Bring Out Your Blogs! (Feed Support Coming)

### DISCUSSION THREAD

**Discussion by StartAutomating** [StartAutomating/Posh#172] [Ideas]
In future versions of Posh, we'd like to add a .Feed to any module (#171).

We'll be adding a simple integrated blog reader to help you find out news about a PowerShell and about a module.

If you have a PowerShell blog (with an RSS feed), please add the RSS URL as a reply to this discussion.

**Comment by HarmVeenstra** [comment]
https://powershellisfun.com/feed/

**Comment by PalmEmanuel** [comment]
Here's mine :) https://pipe.how/index.xml

**Comment by Stephanevg** [comment]
Hi, here is my Blog -> [http://www.powershelldistrict.com/feed](http://www.powershelldistrict.com/feed) :)

**Comment by MikeShepard** [comment]
Mine is here: https://powershellstation.com

**Comment by StartAutomating** [comment]
A few more:

* [PowerShell Team Blog Feed](https://devblogs.microsoft.com/powershell/feed/)
* [The Lonely Administrator](https://jdhitsolutions.com/blog/feed/)
* [The Lazy Admin](https://lazyadmin.nl/feed/)
* [Adam the Automator](https://adamtheautomator.com/feed/)
* [PowerShell Geek](https://www.powershellgeek.com/feed/)
* [Evotec](https://evotec.xyz/feed/)

**Comment by stefanstranger** [comment]
Here is the feed to my blog https://[stefanstranger.github.io/feed](https://stefanstranger.github.io/feed)

**Reply by StartAutomating** [reply]
@stefanstranger  Thanks!  Can you file an issue and give me a PR? (just plop it into the .PSD1 like the others)

**Comment by mdowst** [comment]
https://www.dowst.dev/feed/
